athina/steps/api.py

```
@@ -112,31 +117,44 @@ async def execute_async(self, input_data: Any) -> Union[Dict[str, Any], None]:
+                        json_body = (
+                            json.loads(prepared_body, strict=False)
+                            if prepared_body
+                            else None
+                        )
+                    except json.JSONDecodeError as e:
+                        debug_info = debug_json_structure(prepared_body, e)
+                        return self._create_step_result(
+                            status="error",
+                            data=json.dumps(
+                                {
+                                    "message": f"Failed to parse request body as JSON",
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)